org.jacoco.core/src/org/jacoco/core/instr/Instrumenter.java

```diff
@@ -154,17 +156,19 @@ protected String getCommonSuperClass(final String type1,
 	 */
 	public void instrument(final InputStream input, final OutputStream output,
 			final String name) throws IOException {
+		final byte[] bytes;
```

Comment:
I think we can delegate to the overloaded method here:

	public void instrument(final InputStream input, final OutputStream output,
			final String name) throws IOException {
		output.write(instrument(input, name));
	}